lib/yelp/places.py

```diff
@@ -0,0 +1,71 @@
+import urllib
+import requests
+import pandas as pd
+import sqlalchemy
+
+
+def call_yelp(yelp_api_key, yelp_host, yelp_search_endpoint, categories, location, offset, limit):
+    url = '{0}{1}'.format(yelp_host, urllib.parse.quote(yelp_search_endpoint.encode('utf8')))
```

Comment:
FYI since python 3.5+, you can use another format for string parameter passing:
f"{yelp_host}{urllib.parse.quote(yelp_search_endpoint.encode('utf8'))}"